kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/MessagePublishContext.java

```diff
@@ -60,17 +81,29 @@ public void completed(Exception exception, long ledgerId, long entryId) {
             offsetFuture.completeExceptionally(exception);
         } else {
             if (log.isDebugEnabled()) {
-                log.debug("Success write topic: {}, ledgerId: {}, entryId: {}"
+                log.debug("Success write topic: {}, producerName {} ledgerId: {}, entryId: {}"
                         + " And triggered send callback.",
-                    topic.getName(), ledgerId, entryId);
+                    topic.getName(), this.producerName, ledgerId, entryId);
```

Comment:
It's better to keep the code style. `topic` and `producerName` are both fields of `MessagePublishContext`, but only `producerName` is referenced with `this.` prefix.